test/parallel/test-fs-write-sync-without-offset-and-length.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
Please see https://github.com/nodejs/node/blob/master/test/parallel/test-fs-mkdir.js as an example of what I mean. In some cases, it's overkill to have a bunch of files doing almost exactly the same thing.